PySDM/initialisation/aerosol_composition/dry_aerosol.py

```diff
@@ -71,8 +70,8 @@ def volfrac_just_soluble(self, volfrac: dict, soluble=True):
             x = {k: _masked[k] / _denom for k in self.compounds}
         return x
 
-    # calculate hygroscopicities with different assumptions about solubility
-    def kappa(self, mass_fractions: dict):
+    def kappa(self, mass_fractions: dict, water_molar_volume: float):
```

Comment:
Can we set the default value for this from the constants_default? Otherwise we have to add this input in the other examples (as noted by pylint error).

Reply:
This would be asking for trouble later on - when someone sets some non-default value of `M_v` or `rho_w` and assumes it is propagated, while it would not be so